src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+
+#include <uct/base/uct_cm.h>
+#include <ucs/sys/sock.h>
+#include "rdmacm_def.h"
+
+#define UCT_RDMACM_CM_MAX_CONN_PRIV \
+        (UCT_RDMACM_TCP_PRIV_DATA_LEN) - (sizeof(uct_rdmacm_priv_data_hdr_t))
```

Comment:
why need macro for this?
could it be inline function?

Reply:
in this PR, this value is used once but in the next ones, it will be used several times. 
do you think it's better to use an inline function instead of a define? if yes, can you please explain why?

Reply:
https://stackoverflow.com/a/1137627